src/main/java/de/hs_mannheim/informatik/ct/controller/ContactTracingController.java

```diff
@@ -64,14 +64,43 @@
     private final SimpleDateFormat dateFormatter = new SimpleDateFormat("dd.MM.yyyy");
     private final SimpleDateFormat timeFormatter = new SimpleDateFormat("HH:mm");
 
-    private final List<TracingColumn> tracingColumns = Arrays.asList(
+    private final List<TracingColumn> tracingColumnsStudents = Arrays.asList(
+            new TracingColumn("EMail-Adresse", contact -> contact.getContact().getEmail()),
+            new TracingColumn("Mtknr.", contact -> contact.getContact().getEmail().split("@")[0]),
+            new TracingColumn("Raum/Veranstaltung", Contact::getContactLocation),
+            new TracingColumn("Datum", contact -> dateFormatter.format(contact.getTargetVisit().getStartDate())),
+            new TracingColumn("Anmeldung Ziel", contact -> timeFormatter.format(contact.getTargetVisit().getStartDate())),
+            new TracingColumn("Anmeldung Kontakt", contact -> timeFormatter.format(contact.getContactVisit().getStartDate())),
+            new TracingColumn("Abmeldung Ziel", contact -> timeFormatter.format(contact.getTargetVisit().getEndDate())),
+            new TracingColumn("Abmeldung Kontakt", contact -> timeFormatter.format(contact.getContactVisit().getEndDate())),
+            new TracingColumn("Geburtsdatum", contact -> ""),
```

Comment:
Why is there a date of birth in the table? It is never entered anywhere nor is it a legal requirement, is it? Hence, I would remove it?

Reply:
What I miss in the columns is the phone number though. And I think this is required by law, isn't it?

Reply:
Not shure, mary told me which columns to create, she wanted the date of birth in every sheet and the phone number just for guests. I will adress it in the meeting today.